lib/pbench/test/unit/agent/test_tool_meister.py

```diff
@@ -301,10 +304,11 @@ def getChild(self, arg: Any) -> str:
         assert the_thread.args == (subprocess.PIPE, "parent.child(logger-foo)")
 
     @staticmethod
-    def setup_wait_for_process_tool(logger_name: str) -> Tool:
+    def setup_wait_for_process_tool(logger_name: str, caplog) -> Tool:
```

Comment:
It's too bad that `setup_wait_for_process_tool()` is a `@staticmethod` -- otherwise, it could have gotten `caplog` from `Tool`, and we wouldn't have needed to change all the callers....

Reply:
I agree, it is too bad. =)